From 035183d70fcd75ea3de659ac6350047b419eb600 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Fri, 30 Mar 2018 09:36:19 -0400
Subject: [PATCH] Allow versions of libusb-1.0 newer than 1.0.21 to satisfy the
 dependency

The libusb-1.0 dependency was set to exactly require 1.0.21, which
breaks the build when trying to build against 1.0.22.

Signed-off-by: Neal Gompa <ngompa13@gmail.com>
---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index b0c0a25..3b1982b 100644
--- a/meson.build
+++ b/meson.build
@@ -93,7 +93,7 @@ config_h = configure_file(
 config_h_dir = include_directories('.')
 
 # Required dependencies
-dep_usb = dependency('libusb-1.0', version: '1.0.21')
+dep_usb = dependency('libusb-1.0', version: '>= 1.0.21')
 
 glib_min_version = '>= 2.54.0'
 dep_glib2 = dependency('glib-2.0', version: glib_min_version)

